src/plugins/intel_gpu/tests/functional/behavior/model_cache.cpp

```diff
@@ -304,4 +307,32 @@ INSTANTIATE_TEST_SUITE_P(smoke_CheckWeightlessCacheAccuracyLowPrecision,
                                             ::testing::ValuesIn(low_precision_dtypes)),
                          CheckWeightlessCacheAccuracy::get_test_case_name);
 
+TEST(smoke_CheckWeightlessCacheAccuracy, ConstantFoldingAttrPropagation) {
+    std::vector<uint64_t> data(4, 0);
+    auto constant = std::make_shared<ov::op::v0::Constant>(ov::element::f16, ov::Shape{4}, data);
+    auto attr = ov::WeightlessCacheAttribute(4, 0, ov::element::f16);
```

Comment:
Do I right understand that `WeightlessCacheAttribute::original_size` is "element-size", not "byte-size"?

Reply:
It represents bytes, fixed.